### PR TITLE
[redesign] Mobile dropdown tabs

### LINKED ISSUE #1365: [redesign] Use "dropdown tabs" for mobile screen
Depends on https://github.com/tiagoalvesdulce/pi-ui/pull/131

### PR DESCRIPTION
**Pull request by amassarwi**
- closes #1365

### CODE REVIEW

**Comment by amassarwi** [comment]
I would suggest we make the dropdown-anchor clickable as well before we merge, what do you think ?

**Comment by fernandoabolafio** [comment]
@amassarwi  I agree. The change has to be done on pi-ui and then synced here.

**Comment by amassarwi** [comment]
@fernandoabolafio creating issue in `pi-ui`

**Comment by amassarwi** [comment]
waiting on [tiagoalvesdulce/pi-ui#141](https://github.com/tiagoalvesdulce/pi-ui/issues/141)

**Comment by amassarwi** [comment]
Updated tabs in user page as well, the dropdown arrow is not aligned correctly there, this may need more changes in `pi-ui`

**Comment by amassarwi** [comment]
> Updated tabs in user page as well, the dropdown arrow is not aligned correctly there, this may need more changes in `pi-ui`

Tackled :tada:

File: README.md [hunk 1]
```diff
@@ -33,13 +33,22 @@ Politeiagui uses node.js and the yarn package manager.
     
     `yarn && yarn start`
 
+    **Running v2 [redisgn]**
```
Comment (fernandoabolafio) [inline README.md]: Can you move the Docs changes to another PR? I want to have it done separately so we can craft it carefully.  Just created an issue to track it: https://github.com/decred/politeiagui/issues/1369

Comment (amassarwi) [reply README.md]: Sure

**Comment by amassarwi** [comment]
@fernandoabolafio 

**Current state locally:**
image
